### PR TITLE
[dv/full chip] Update chip testplan

### PR DESCRIPTION
**Pull request by matutem**
Signed-off-by: Guillermo Maturana <maturana@google.com>

### CODE REVIEW

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -664,11 +664,9 @@
       name: chip_sw_clk_idle_trans
       desc: '''Verify the ability to turn off the transactional clock via SW.
 
-            Ensure that activity in any of the IPs running on this clock prevents the clock from
-            actually being turned off until the activity is completed, at which point the clock is
-            turned off.  Verify it is off via spinwait in hints_status CSR, and then check the state
-            of the transactional unit to correspond to a completed activity.  Verify that turning
-            off this clock does not affect the other derived clocks.
+            Ensure that the clock to transactional units can be turned off regardless of any
```
Comment (sriyerg) [inline hw/top_earlgrey/data/chip_testplan.hjson:667]: Note that these are already X-ref'ed with other IP tests that generated the idle. So the cost of accomplishing this testpoint is zero.

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 2]
```diff
@@ -704,21 +702,33 @@
...
+      tests: []
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_sw
+      desc: '''Verify SW causes the clkmgr requests ext clk src during certain LC states.
+
+            In RMA and TEST_UNLOCKED lc states the external clock is enabled in response to
+            `extclk_ctrl.sel` CSR writes. In addition `extclk_ctrl.step_down` CSR causes the divided
+            clocks to step down. It may be best to verify this via SVA, unless we implement clock
+            cycle counters.
```
[the remark this answers is not in the document]
Comment (matutem) [reply hw/top_earlgrey/data/chip_testplan.hjson:731]: Done.

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 3]
```diff
@@ -664,11 +664,9 @@
...
       desc: '''Verify the ability to turn off the transactional clock via SW.
 
-            Ensure that activity in any of the IPs running on this clock prevents the clock from
-            actually being turned off until the activity is completed, at which point the clock is
-            turned off.  Verify it is off via spinwait in hints_status CSR, and then check the state
-            of the transactional unit to correspond to a completed activity.  Verify that turning
-            off this clock does not affect the other derived clocks.
+            Ensure that the clock to transactional units can be turned off regardless of any
+            activity in the transactional IP.  Verify it is off via spinwait in hints_status CSR.
+            Verify that turning off this clock does not affect the other derived clocks.
```
[the remark this answers is not in the document]
Comment (matutem) [reply hw/top_earlgrey/data/chip_testplan.hjson:669]: Most of these tests check that as a side effect: the processor keeps running and checks things are off, and then turns these clocks back on. That wouldn't work if the processor was out or there was a reset.

[the remark this answers is not in the document]
Comment (tjaychen) [reply hw/top_earlgrey/data/chip_testplan.hjson:669]: sounds good!

**Pull request merged**